packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/GraphTicks.test.js

```diff
@@ -12,33 +12,40 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { shallow } from 'enzyme';
+import { render, screen, within } from '@testing-library/react';
 import React from 'react';
 
 import GraphTicks from './GraphTicks';
 
-describe('<GraphTicks>', () => {
+const setup = (propOverrides) => {
   const defaultProps = {
     items: [
       { valueWidth: 100, valueOffset: 25, serviceName: 'a' },
       { valueWidth: 100, valueOffset: 50, serviceName: 'b' },
     ],
     valueWidth: 200,
     numTicks: 4,
+    ...propOverrides,
   };
 
-  let ticksG;
-
-  beforeEach(() => {
-    const wrapper = shallow(<GraphTicks {...defaultProps} />);
-    ticksG = wrapper.find('[data-test="ticks"]');
-  });
+  return render(<GraphTicks {...defaultProps} />);
+};
 
+describe('GraphTicks tests', () => {
   it('creates a <g> for ticks', () => {
-    expect(ticksG.length).toBe(1);
+    const consoleSpy = jest.spyOn(console, 'error').mockImplementation();
+    setup();
+
+    // Expect ----->  Warning: The tag <line> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.
+    expect(consoleSpy).toHaveBeenCalledTimes(2);
```

Comment:
Is there a better way to do this? Or is this warning 'expected' ?

Reply:
I'd say if you need to mock any of the console methods for the tests to pass, it's a 99% chance there's smth wrong with either the tests or component you're testing. In this case you're trying to render svg's `g` tag outside of svg, which causes the error. To make this work, wrap the tested component in `svg`: 
```
  return render(
    <svg>
      <GraphTicks {...defaultProps} />
    </svg>
  );
```

Reply:
Yeah, I didn't want to leave it like that.  I appreciate the explanation, thanks for the help. 🙂